lib/core/form/components/widgets/core/form-field.model.ts

```diff
@@ -403,6 +396,12 @@ export class FormFieldModel extends FormWidgetModel {
             case FormFieldTypes.BOOLEAN:
                 this.form.values[this.id] = (this.value !== null && this.value !== undefined) ? this.value : false;
                 break;
+            case FormFieldTypes.PEOPLE:
```

Comment:
the code is the same for BOOLEAN/PEOPLE/FUNCTIONAL_GROUP? use three `case` blocks together, i.e https://stackoverflow.com/a/13207939/1716421

Reply:
thanks. my mistake!